logging/google/cloud/logging/handlers/_helpers.py

```diff
@@ -16,6 +16,17 @@
 
 import math
 import json
+import sys
+
+try:
+    import flask
+except ImportError:
+    flask = None
+
```

Comment:
The flask.request.headers is used when getting trace_id, so this import should be kept here.